configs/default/ANYL-MERCURYG4.config

```diff
@@ -0,0 +1,142 @@
+# Betaflight / STM32G47X (SG47) 4.4.0 Feb 25 2023 / 03:35:00 (8d4f005) MSP API: 1.44
+
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO_DPS310
+#define USE_FLASH_M25P16
+#define USE_GPS
+//TODO#define SYSTEM_HSE_MHZ 16
```

Comment:
```suggestion
```
Is this comment needed? Looks like we could just remove it from the "official" config

Reply:
This is in line with some recent changes for an upcoming change.